tests/test_devices.py

```diff
@@ -1135,3 +1135,23 @@ async def test_led_status_request_exception(
     )
     with pytest.raises(AttributeError):
         DeviceSetLedStatus.create(device, **data)
+
+
+@pytest.mark.parametrize(("device_payload"), [[PDU_PRO]])
```

Comment:
That is a PDU, shouldn't the WAN sensor be on a dream machine or equal device? Move the data to a more correct fixture please.

Reply:
Not sure what a PDU was, it looked (based on the data) like something where it did fit. 

I don't see a dream machine. 
Will a USG be a better fit or should I create a dream machine fixture?

Reply:
Pdu is a power distribution unit. It's fine to put in with the usg